examples/jaxsim_as_physics_engine.ipynb

```diff
@@ -75,242 +71,136 @@
+    "The `JaxSimModel` object contains the simulation time step, the integrator and the contact model.\n",
+    "To see the advanced usage, check the advanced example, where you will see how to pass explicitly an integrator class and state to the `model` object and how to change the contact model."
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {
-    "id": "FJF-HoWaiK9J"
-   },
+   "metadata": {},
    "source": [
-    "### Select the contact model\n",
-    "\n",
-    "JaxSim offers several contact models, with the default being the non-linear Hunt/Crossley soft contact model. This model supports stick/slip transitions and fully accounts for friction cones.\n",
-    "\n",
-    "While it is faster than other models, it requires careful parameter tuning and may need a small time step $\\Delta t$, unless a variable-step integrator is used.\n",
-    "\n"
+    "### Create the model "
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "id": "VAaitHRKjnwc"
+    "id": "etQ577cFaTHA"
    },
    "outputs": [],
    "source": [
-    "import jaxsim\n",
-    "\n",
-    "# Operate on a copy of the model.\n",
-    "# When validate=True, this context manager ensures that the PyTree structure\n",
-    "# of the object is not altered. This is a nice feature of JaxSim to spot\n",
-    "# earlier user logic that might trigger unwanted JIT recompilations.\n",
-    "# In this case, we need to disable validation since PyTree structure might\n",
-    "# change if you use a contact model different from the default.\n",
-    "with model.editable(validate=False) as model:\n",
-    "\n",
-    "    # The SoftContacts class can be replaced with a different contact model.\n",
-    "    model.contact_model = jaxsim.rbda.contacts.SoftContacts.build()\n",
-    "\n",
-    "# JaxSim provides the following helper that estimates good contact\n",
-    "# parameters. While they might not be optimal, usually are a good\n",
-    "# starting point. Users are encouraged to fine-tune them.\n",
-    "contacts_params = js.contact.estimate_good_contact_parameters(\n",
-    "    model=model,\n",
-    "    number_of_active_collidable_points_steady_state=4,\n",
-    "    max_penetration=0.001,\n",
+    "#  Create the JaxSim model.\n",
+    "url = \"https://raw.githubusercontent.com/icub-tech-iit/ergocub-software/refs/heads/master/urdf/ergoCub/robots/ergoCubSN001/model.urdf\"\n",
+    "# Create a temporary file\n",
+    "with tempfile.NamedTemporaryFile(mode=\"w+\", delete=False) as urdf_robot_file:\n",
+    "    # Retrieve the file\n",
+    "    urllib.request.urlretrieve(url, urdf_robot_file.name)\n",
+    "\n",
+    "# print(urllib.request.urlretrieve(url, urdf_robot_file.name))\n",
+    "model_description_path = pathlib.Path(urdf_robot_file.name)\n",
+    "full_model = js.model.JaxSimModel.build_from_model_description(\n",
+    "    model_description=model_description_path,\n",
+    "    time_step=0.0001,\n",
+    "    is_urdf=True\n",
```

Comment:
This is actually needed, indeed since the file is taken from a temp file, the extension .urdf is not there and the file is not correctly recognised as an urdf

Reply:
Alright, thanks for the explanation!